test/e2e/iam/orgs_test.go

```diff
@@ -45,12 +46,18 @@ func TestOrgs(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 		a := assert.New(t)
 		if a.NoError(err, string(resp)) {
-			var orgs mongodbatlas.Organizations
+			var orgs interface{}
 			if err := json.Unmarshal(resp, &orgs); err != nil {
 				t.Fatalf("unexpected error: %v", err)
 			}
-			a.NotEmpty(orgs.Results)
-			orgID = orgs.Results[0].ID
+			switch r := orgs.(type) {
+			case *atlasv2.PaginatedOrganization:
+				a.NotEmpty(r.Results)
+				orgID = *r.Results[0].Id
+			case *atlas.Organizations:
+				a.NotEmpty(r.Results)
+				orgID = r.Results[0].ID
+			}
```

Comment:
FYI you don't have to worry about casting here since the json is equivalent you can use any struct that knows how to deserialize the json and both work, pretty sure we have done this in multiple test where we deserialize as atlas specific struct if we don't need any OM specific field